python/benchmarks/oxford_pet/datagen.py

```diff
@@ -106,6 +105,7 @@ def _convert(obj_list):
             return dict(zip(keys, [[]] * len(keys)))
 
         with_xmls["object"] = with_xmls["object"].apply(_convert)
+        with_xmls.reset_index(names=["_pk"], inplace=True)
```

Comment:
with_xmls is the result of pd.concat which can sometimes mess with the index.
IIUC, here we just want an incrementing sequence of integers right? let's be defensive and just do:

```python
(with_xmls.reset_index(drop=True)
.reset_index(names=["_pk"], inplace=True))
```

https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.reset_index.html